### PR TITLE
Add oryp10

### PR DESCRIPTION
**Pull request by jacobgkau**
This adds the Oryx Pro 10 to tech docs.

The repair and internal overview photos have been reshot with an oryp10 unit. The external overview page keeps the oryp9 port diagrams (marketing did not reshoot port photos because the chassis did not change); while repair and internal overview photos have switched from 17" to 15", port diagrams stay at the 17" model because marketing did not shoot closed left/right photos of the 15" oryp9.

The main spec updates are the switch from DDR4 3200MHz RAM to DDR5 4800MHz RAM and the addition of a 15" OLED display option. The repairs page reflects that heat-conductive metal is only used to cool the GPU (the entire CPU uses regular thermal paste.)

I tried to add verbiage to the repairs page warning about the difficulty and risk of removing the keyboard. I currently do not have anything about using rubbing alcohol to dissolve the adhesive (that seems like a risky thing to recommend.)

### CODE REVIEW

File: src/models/oryp10/external-overview.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+![Keyboard & Touchpad](./img/keyboard-touchpad.webp)
+
+## Materials:
+
+|Part                              |Material                |
+|----------------------------------|------------------------|
+|LCD back cover (lid)              |Aluminum                |
+|LCD front cover (bezel)           |Plastic                 |
+|Top case (palm rests/port covers) |Aluminum                |
```
Comment (leviport) [inline src/models/oryp10/external-overview.md:29]: The top case is plastic on oryp10 and oryp9. I think I missed this on the oryp9 docs.

Comment (jacobgkau) [reply src/models/oryp10/external-overview.md]: Yikes. Fixed with 8e530b15676b8b50667f5f6813702e0d199dc4af.

**Review by leviport — APPROVED** [review]
Looks great! I think this is ready to merge as soon as the product launches.

**Comment by jacobgkau** [comment]
Looks like the CI build is failing for some reason:

```
   Compiling mdbook v0.4.18
error[E0597]: `local_ctx` does not live long enough
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/mdbook-0.4.18/src/renderer/html_handlebars/helpers/navigation.rs:154:25
    |
154 |             t.render(r, &local_ctx, &mut local_rc, out)
    |                         ^^^^^^^^^^ borrowed value does not live long enough
155 |         })?;
    |         -
    |         |
    |         `local_ctx` dropped here while still borrowed
    |         borrow might be used here, when `local_rc` is dropped and runs the destructor for type `handlebars::RenderContext<'_, '_>`
    |
    = note: values in a scope are dropped in the opposite order they are defined

For more information about this error, try `rustc --explain E0597`.
error: could not compile `mdbook` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `mdbook v0.4.18`, intermediate artifacts can be found at `/tmp/cargo-install8itaP8`
Error: Process completed with exit code 101.
```

**Comment by jacobgkau** [comment]
Looks like this was worked around in mdbook: https://github.com/rust-lang/mdBook/issues/1860 

Although it sounds like it was caused by a Rust update: https://github.com/rust-lang/rust/issues/99591 

Not sure why this is becoming a problem for us right now, but I'll try updating mdbook to the latest version.

**Review by leviport — APPROVED** [review]

**Pull request merged**